business_requirement_deliverable_default/models/business_requirement_deliverable_default.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class BusinessRequirementDeliverable(models.Model):
+    _inherit = "business.requirement.deliverable"
+
+    def _prepare_resource_lines(self):
+        return self.product_id.sudo().resource_lines.copy_data()
+
+    @api.onchange('product_id')
+    def product_id_change(self):
+        super(BusinessRequirementDeliverable, self).product_id_change()
+        product = self.product_id
+        if product:
+            rl_data = self._prepare_resource_lines()
+            br_resource_obj = self.env['business.requirement.resource']
+            for item in rl_data:
+                br_resource_obj += br_resource_obj.new(item)
+            self.resource_ids = br_resource_obj
+
+
+class ProductTemplate(models.Model):
```

Comment:
@elicoidal Sorry, I did that in XML but now I have done that in py as well.
Thanks for drawing the attention.